src/libraries/System.Net.Requests/src/System/Net/FtpControlStream.cs

```diff
@@ -211,7 +227,7 @@ protected override void ClearState()
 
             _dataSocket = null;
             _passiveEndPoint = null;
-            _tlsStream = null;
+            _sslStream = null;
```

Comment:
```suggestion
            _sslStream = null;
            _dataStream = null;
```

I don't think it functionally matters though